org.kde.kruler.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "org.kde.kruler",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kruler",
+    "rename-icon": "kruler",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        {
+            "name": "kruler",
+            "buildsystem": "cmake-ninja",
+            "sources": [
```

Comment:
The formatting is a little bit off here.

Reply:
Lines 15-18 look okay to me; if there's something specific you'd like to suggest, please do.

I did find a formatting issue in the lines below. I've updated them : https://github.com/flathub/flathub/pull/2750/commits/71b46d00f8fb5a2568f2ae04c9972df8ec6171c2